### DISCUSSION TITLE
go get ariga.io/atlas/cmd/atlas@v0.4.2 is returning an error

### DISCUSSION THREAD

**Discussion by manolo988** [ariga/atlas#894] [Q&A] [answered]
panic: internal error: can't find reason for requirement on github.com/mattn/go-colorable@v0.1.9

v0.4.1 works

**Comment by a8m** [comment, marked as answer]
Hey @manolo988 and sorry for the delayed response. 
The latest version should work now 🙏

**Reply by manolo988** [reply]
Thanks
